examples/Batch/WordCountFSharp/WordCountExample.fs

```diff
@@ -0,0 +1,34 @@
+﻿open Microsoft.Spark.CSharp.Core
+open Microsoft.Spark.CSharp.Services
+open System.Reflection
+open System.Collections.Generic
+
+[<EntryPoint>]
+let main = function
+    | [| filePath |] ->
+        let logger =
+            LoggerServiceFactory.SetLoggerService Log4NetLoggerService.Instance
+            LoggerServiceFactory.GetLogger (MethodInfo.GetCurrentMethod().DeclaringType)
+        
+        let sparkContext = SparkConf().SetAppName "MobiusWordCount" |> SparkContext
+        logger.LogInfo (sprintf "Reading from file %s" filePath)
+
+        try
```

Comment:
Not really - you need to create a standalone executable which is then launched from Spark (at least, that's the experience I've had thus far).

Try/catch etc. - again, this is essentially a like-for-like from the C# example so yes it's as much as showing that as anything else.